src/guillo-gmi/reducers/guillotina.js

```diff
@@ -49,7 +49,11 @@ export function guillotinaReducer(state, action) {
       return { ...state, action: { action: undefined, params: undefined } }
 
     case 'REFRESH':
-      return { ...state, refresh: Math.random(), loading: true }
+      return {
+        ...state,
+        refresh: Math.random(),
```

Comment:
Això potser ho canviaria per:

```suggestion
        refresh: Date.now(),
```

Per molt petita que sigui la probabilitat, hi ha la probabilitat de que es repeteixi el valor del `Math.random()` i no faci el refresh...